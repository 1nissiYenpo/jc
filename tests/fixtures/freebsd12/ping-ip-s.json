{"destination_ip": "127.0.0.1", "data_bytes": 40, "pattern": null, "destination": "127.0.0.1", "packets_transmitted": 3, "packets_received": 3, "packet_loss_percent": 0.0, "round_trip_ms_min": 0.024, "round_trip_ms_avg": 0.032, "round_trip_ms_max": 0.036, "round_trip_ms_stddev": 0.006, "responses": [{"type": "reply", "bytes": 48, "response_ip": "127.0.0.1", "icmp_seq": 0, "ttl": 64, "time_ms": 0.024}, {"type": "reply", "bytes": 48, "response_ip": "127.0.0.1", "icmp_seq": 1, "ttl": 64, "time_ms": 0.036}, {"type": "reply", "bytes": 48, "response_ip": "127.0.0.1", "icmp_seq": 2, "ttl": 64, "time_ms": 0.036}]}
