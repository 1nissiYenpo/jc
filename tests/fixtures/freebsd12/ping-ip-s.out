PING 127.0.0.1 (127.0.0.1): 40 data bytes
48 bytes from 127.0.0.1: icmp_seq=0 ttl=64 time=0.024 ms
48 bytes from 127.0.0.1: icmp_seq=1 ttl=64 time=0.036 ms
48 bytes from 127.0.0.1: icmp_seq=2 ttl=64 time=0.036 ms

--- 127.0.0.1 ping statistics ---
3 packets transmitted, 3 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 0.024/0.032/0.036/0.006 ms
